README.md

```diff
@@ -6,22 +6,52 @@
 
 [![codecov.io](http://codecov.io/github/ahnlabb/BioformatsLoader.jl/coverage.svg?branch=master)](http://codecov.io/github/ahnlabb/BioformatsLoader.jl?branch=master)
 
-## Usage
+## Dependencies
 
-Requires `bioformats_package.jar`
+Depends on `bioformats_package.jar`
+
+## Install
+
+```julia
+Pkg.add("https://github.com/ahnlabb/BioformatsLoader.jl") # Change if forked
```

Comment:
This doesn't work when using the `Pkg.add` function, it needs a `PackageSpec` when adding a package not in a registry. The pkg REPL/pkg string literal is more flexible so you can use `] add https://github.com/ahnlabb/BioformatsLoader.jl` or `using Pkg; pkg"add https://github.com/ahnlabb/BioformatsLoader.jl"` otherwise you need to use:
```julia
import Pkg
Pkg.add(Pkg.PackageSpec(url="https://github.com/ahnlabb/BioformatsLoader.jl"))
```

Reply:
If this package is useful to many people the sensible thing would of course be for me to just try to get it into the julia registry.

Reply:
We definitely should get it into the registry. I'm glad that you responded.

I've been working on JavaCall.jl upstream since the stack conventions in 1.1 broke a few things making it difficult to work with JavaCall. Now we have the environmental variable `JULIA_COPY_STACKS=1` which fixes things comprehensively. JavaCall now warns about this.

JavaCall.jl also works ok without JULIA_COPY_STACKS if it is being used on the root `Task`. As of Julia 1.5, REPL commands are now executed on the root `Task` by default.

Given that the above fixes, I think we should get more Java interop packages into the registry.

Reply:
Maybe `PackageSpec` is not needed on newer Julia. I seem to recall that working just fine. https://julialang.github.io/Pkg.jl/v1/managing-packages/#Adding-packages-1

Reply:
> I've been working on JavaCall.jl upstream since the stack conventions in 1.1 broke a few things making it difficult to work with JavaCall. Now we have the environmental variable `JULIA_COPY_STACKS=1` which fixes things comprehensively. JavaCall now warns about this.

That's great to hear!

> Maybe `PackageSpec` is not needed on newer Julia. I seem to recall that working just fine. https://julialang.github.io/Pkg.jl/v1/managing-packages/#Adding-packages-1

I might be wrong but I don't think that should work or has worked, it errors on my machine on 1.4.0 and it used to be `Pkg.clone` for git repos. See JuliaLang/Pkg.jl#629.

Reply:
```
Pkg.add(url="https://github.com/JuliaLang/Example.jl", rev="master") # From url to remote gitrepo
```
This might be a more compact syntax: https://github.com/JuliaLang/Pkg.jl/blob/master/src/Pkg.jl#L126

Reply:
Fixed the syntax